src/platforms/browser/WebPlatform.mjs

```diff
@@ -106,14 +106,15 @@ export default class WebPlatform {
     }
 
     uploadCompressedGlTexture(gl, textureSource, source, options) {
+        const view = !source.pvr ? new DataView(source.mipmaps[0]) : source.mipmaps[0];
```

Comment:
Are more different types of sources besides `pvr = true` expected in the future? If so, then we could make this a bit more 'future proof'